cli/module_generate/_templates/python/src/tmpl-main.py

```diff
@@ -3,7 +3,7 @@
 try:
     from models.{{ .ModelSnake }} import {{ .ModelPascal }}
 except ModuleNotFoundError:
-    # when running as local module with run.sh
+    # when running as local module
     from .models.{{ .ModelSnake }} import {{ .ModelPascal }}
```

Comment:
Probably something that @njooma should decide more than me, but can we get rid of the double import and only import once? If the generator is focused on modules that will be used with cloud build and pyinstaller, I think we only need to import one way. Expert users who want to do something different will know how to change this and will come up with their own custom code structure and execution logic.